uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java

```diff
@@ -313,6 +314,24 @@ public static void handleButtonClick(@NonNull final String resourceId) {
         }
     }
 
+    /**
+     * Clicks the button element associated to the supplied resource id with a longer timeout.
+     * Creating another method to use the longer timeout here, some ui interactions take longer than expected.
+     *
+     * @param resourceId the resource id of the button to click
+     */
+    public static void handleButtonClickLongTimeout(@NonNull final String resourceId) {
```

Comment:
We can probably write a method as follows:

```java
public static void handleButtonClick(@NonNull final String resourceId, long timeOut);
```

Something like that. this would allow it to be more easily configurable in the future.

Reply:
Yes, that is a much better solution.